library/src/fy_library/mixins/property/mixin_imports/using_parsed_fy_py_file_fy.py

```diff
@@ -54,29 +55,17 @@ def _mixin_imports(self) -> List[str]:
         # fy:end <<<===
         match self._parsed_fy_py_file.file_type:
             case ParsedFyPyFileKind.FLOW:
-                property_setters = [
-                    property_mixin
-                    for property_mixin in cast(
+                property_setters = FlowImports_Flow(
+                    property_mixins=cast(
                         ParsedFlowFyPyFile, self._parsed_fy_py_file
-                    ).template_model.properties
-                    if property_mixin.implementation_name.snake_case == "setter"
-                ]
+                    ).template_model.properties,
+                    parsed_fy_py_files_map_by_key=self._parsed_fy_py_files_map_by_key,
+                )()
                 import_any = ["from typing import Any"] if property_setters else []
-                user_imports: Set[str] = set()
-                for property_setter in property_setters:
-                    user_imports.update(
-                        [
-                            user_import
-                            for user_import in self._parsed_fy_py_files_map_by_key[
-                                property_setter.property_name.snake_case
-                            ].user_imports.split("\n")
-                            if user_import != ""
-                        ]
-                    )
 
                 mixin_imports = (
                     import_any
-                    + list(user_imports)
+                    + list(property_setters)
```

Comment:
I checked, it dissapeared at the end, but in any case, no need for `list` conversion here, it is already a list.